chart/tests/test_flower.py

```diff
@@ -22,12 +22,21 @@
 from tests.helm_template_generator import render_chart
 
 
-class FlowerAuthorizationTest(unittest.TestCase):
+class FlowerTest(unittest.TestCase):
```

Comment:
given that airflow uses pytest shouldn't the class be named TestFlower?

Reply:
Your are right.